tools/subgraph-example/README.md

```diff
@@ -1,35 +1,58 @@
 # Subgraph example
 
-Simple subgraph indexing the `GreeterSet` event of emitted by the [modified] Greeter contract
+Hardhat based project containing example subgraph indexing the following contracts and events:
+
+ExampleERC721 -> Transfer
```

Comment:
```suggestion
ExampleERC721 -> Mint
```

Reply:
When minting an NFT it still emits a `Transfer` event where `from` is the zero address

Reply:
Ah you're correct, this is highlighting events.